docker-compose.dev.yaml

```
@@ -1,10 +1,69 @@
+version: '3.8'
+
+name: NSHM
+
 services:
+  # Database Services
+  postgres:
+    image: postgres
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_USER=${POSTGRES_USERNAME}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+    volumes:
+      - ./services/account/database:/docker-entrypoint-initdb.d
+    restart: always
+
+  mongo_item:
+    image: mongo
+    ports:
+      - 27017:27017
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=${ITEM_SERVICE_MONGO_DB_USER}
+      - MONGO_INITDB_ROOT_PASSWORD=${ITEM_SERVICE_MONGO_DB_PASS}
+      - MONGO_INITDB_DATABASE=${ITEM_SERVICE_MONGO_DB}
+    volumes:
+      - ./services/item/mongodb:/data/db
```

Rather than mounting `/data/db` to host machine's files, we should mount `/docker-entrypoint-initdb.d` to `./services/item/database/dev`
```suggestion
      - ./services/item/database/dev:/docker-entrypoint-initdb.d
```